label.scss

```
@@ -0,0 +1,23 @@
+@use 'node_modules/@mapsindoors/midt/font';
+@use 'node_modules/@mapsindoors/midt/margin';
+@use 'node_modules/@mapsindoors/midt/color';
+
+@use "node_modules/@mapsindoors/midt/build/scss/variables" as variables;
+
+.mi-label {
+    font-style: normal;
+    @include font.size(medium);
+    @include font.weight(large);
+    @include color.black(base);
+    @include margin.bottom(medium);
+
+    &>* {
```

My preference would be
```suggestion
    & > * {
```
I find that easier to read. Take it or leave it

---

I know, but I haven't figured how to configure my sass formatter  🤷